docs/spec/lightclient/detection/detection.md

```diff
@@ -136,10 +316,18 @@ Let *a*, *b*, *c*, be light blocks and *t* a time, we define
 
 implies *b = c*.
 
+
+#### **[TMBC-SIGN-FORK.1]**
+
 If there exists three light blocks a, b, and c, with 
 *sign-skip-match(a,b,c,t) =
-false* then we have a *slashable fork*.
+false* then we have a *slashable fork*.  
```

Comment:
I think we need to be more precise here: sign-skip-match(a,b,c,t) = false might also hold when sequ-rooted(a) = false. So we should probably give the full condition and not negation of the previous one.

Reply:
I think we are precise: Sign-skip-match(a,b,c,t) is defined as implication, A -> B with

A = 
   - *sequ-rooted(a)* and
   - *b.Header.Height = c.Header.Height* and
   - *skip-root(a,b,t)* and
   - *skip-root(a,c,t)*

B = "b = c"

So if sequ-rooted(a) = false this means A is false and so the whole implication must be true, and thus sign-skip-match(a,b,c,t) = true

Reply:
Let's add a discussion with this explanation into the text.